src/github.com/couchbaselabs/sync_gateway/rest/routing.go

```diff
@@ -168,6 +168,22 @@ func CreateAdminHandler(sc *ServerContext) http.Handler {
 	r.Handle(kDebugURLPathPrefix,
 		makeHandler(sc, adminPrivs, (*handler).handleExpvar)).Methods("GET")
 
+	// Debugging handlers
+	r.Handle("/_debug/pprof/goroutine",
+		makeHandler(sc, adminPrivs, (*handler).handlePprofGoroutine)).Methods("GET", "POST")
+	r.Handle("/_debug/pprof/cmdline",
+		makeHandler(sc, adminPrivs, (*handler).handlePprofCmdline)).Methods("GET", "POST")
+	r.Handle("/_debug/pprof/symbol",
```

Comment:
This endpoint ("/_debug/pprof/symbol") needs to handle POST requests.  I wasn't sure what other endpoints also needed this, so I just added "GET", "POST" to all of them.